engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/task/UserTaskProcessor.java

```diff
@@ -42,7 +57,18 @@ protected boolean isJobBehavior(
   @Override
   protected void onActivateInternal(
       final ExecutableUserTask element, final BpmnElementContext context) {
-    // TODO will be added with https://github.com/camunda/zeebe/issues/15275
+    variableMappingBehavior
+        .applyInputMappings(context, element)
+        .flatMap(ok -> userTaskBehavior.evaluateUserTaskExpressions(element, context))
+        .flatMap(j -> eventSubscriptionBehavior.subscribeToEvents(element, context).map(ok -> j))
+        .ifRightOrLeft(
+            userTaskProperties -> {
+              final var userTaskKey =
+                  userTaskBehavior.createNewUserTask(context, element, userTaskProperties);
+              userTaskBehavior.userTaskCreated(userTaskKey, context, element, userTaskProperties);
```

Comment:
:thought_balloon: Similar to what I wrote about the User task event appliers, the logic here will diverge with later iterations since we will have additional operations between the `UserTask.CREATING` and `UserTask.CREATED` events.